pkg/deployments/README.md

```diff
@@ -102,6 +102,7 @@ Returns an object with all contracts from a deployment and their addresses.
 | Batch Relayer V4                                        | [`20220916-batch-relayer-v4`](./tasks/20220916-batch-relayer-v4)                                     |
 | Managed Pool                                            | [`20221021-managed-pool`](./tasks/20221021-managed-pool)                                             |
 | Protocol Fee Splitter                                   | [`20221027-fee-splitter`](./tasks/20221027-fee-splitter)                                             |
+| L2 Gauge Checkpointer                                   | [`20221101-l2-gauge-checkpointer`](./tasks/20221101-l2-gauge-checkpointer)                           |
```

Comment:
Can you also update the changelog?

Reply:
Both of deployments and interfaces (which should have been done in #1570)

Reply:
Done.